hasher-matcher-actioner/hmalib/common/writebacker_models.py

```diff
@@ -180,8 +193,8 @@ def _writeback_impl(self, writeback_signal: BankedSignal) -> str:
             "review_status": "REVIEWED_MANUALLY",
             "status": "MALICIOUS",
         }
-        print(descriptors)
 
+        # If we already have a descriptor we can copy it and upload make sure to never expire it
         if my_descriptor:
```

Comment:
nit: is this a function call or a variable/property? (probably alright to leave if you rename `self.my_descriptor(...)`)